lib/bugsnex/api.ex

```diff
@@ -15,6 +15,11 @@ defmodule Bugsnex.Api do
     post("/", body)
   end
 
+  def send_deploy(deploy) do
+    body = Poison.encode!(deploy)
+    post("/deploy", body)
```

Comment:
from [the bugsnag deploy documentation](https://bugsnag.com/docs/deploy-tracking-api):

> If you choose to POST a JSON object, make sure you set the HTTP Content-Type header to be application/json.

we don't set that header, right?